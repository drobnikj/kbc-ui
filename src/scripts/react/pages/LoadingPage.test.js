import React from 'react';

import LoadingPage from './LoadingPage';

describe('<LoadingPage />', function() {
  it('should render with no props', function() {
    shallowSnapshot(
      <LoadingPage />
    );
  });
});
