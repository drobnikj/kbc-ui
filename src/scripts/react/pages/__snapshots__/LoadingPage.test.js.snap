exports[`<LoadingPage /> should render with no props 1`] = `
<div
  className="container-fluid">
  <div
    className="kbc-main-content">
    <div
      className="kbc-main-loader">
      <span>
        Loading ...
      </span>
    </div>
  </div>
</div>
`;
